### PR TITLE
Mute neuro-kill errors

### PR DESCRIPTION
**Pull request by ayushkovskiy**
Fixes https://github.com/neuromation/cookiecutter-neuro-project/issues/71

### CODE REVIEW

File: {{cookiecutter.project_slug}}/Makefile [hunk 1]
```diff
@@ -55,7 +55,7 @@ help:
 
 .PHONY: setup
 setup: ### Setup remote environment
-	neuro kill $(SETUP_NAME)
+	neuro kill $(SETUP_NAME) >/dev/null 2>&1
```
Comment (anayden) [inline {{cookiecutter.project_slug}}/Makefile]: I would extract `neuro kill >/dev/null 2>&1` to a variable and reuse everywhere. Something like `$(NEURO_KILL) $(SETUP_NAME)`, just like we do with `neuro cp`

Comment (mariyadavydova) [reply {{cookiecutter.project_slug}}/Makefile]: @anayden, is it correct that
```
neuro kill setup-project >/dev/null 2>&1
```
and
```
neuro kill >/dev/null 2>&1 setup-project
```
work the same?

Comment (anayden) [reply {{cookiecutter.project_slug}}/Makefile]: As far as I know — yes

Comment (ayushkovskiy) [reply {{cookiecutter.project_slug}}/Makefile]: > I would extract neuro kill >/dev/null 2>&1 to a variable and reuse everywhere.

Is this kind of meta-programming possible in the syntaxes of Bash and Makefile?

Comment (mariyadavydova) [reply {{cookiecutter.project_slug}}/Makefile]: I would rather call it Bash magic... This won't work on Windows, BTW.

">/dev/null" redirects all output to nowhere 

"2>&1" redirects stderr to stdout

Alex Davydow recommends putting these instructions before "neuro kill".

Comment (ayushkovskiy) [reply {{cookiecutter.project_slug}}/Makefile]: I know, but I something like this won't work:
```
redirect=">/dev/null 2>&1"
neuro kill shomething $redirect
$redirect neuro kill something
```
@anayden what was your idea of shortcutting this syntax?

Comment (ayushkovskiy) [reply {{cookiecutter.project_slug}}/Makefile]: > Alex Davydow recommends putting these instructions before "neuro kill".

Why? I have never seen it, and, to be honest, `>/dev/null 2>&1 neuro kill $(SETUP_NAME)` looks much more scary for me than `neuro kill $(SETUP_NAME) >/dev/null 2>&1`.

> I would extract neuro kill >/dev/null 2>&1 to a variable and reuse everywhere.

I analyzed the issue and came to that this has to be done only once for `make setup` -- so this is not a problem, even though Makefile supports the following syntax: 
```
MUTE:=">/dev/null 2>&1"
setup:
    ${MUTE} neuro kill job-name
```

**Pull request merged**